spec/features/department_deletion_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe 'deleting users' do
+  before :each do
+    when_current_user_is :staff, integration: true
+    @dept = create :department
```

Comment:
I'd prefer, before your `before :each` block:

```ruby
let(:department) { create :department }
```

... and then you can refer to it everywhere as a local variable.

In general, prefer `let` blocks to instance variables (source: https://github.com/nevir/rubocop-rspec/blob/master/lib/rubocop/cop/rspec/instance_variable.rb#L6-L20)